haproxy-tools.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency('net-scp')
   gem.add_dependency('treetop')
-  gem.add_development_dependency('rake')
+  gem.add_development_dependency('rake', '< 11.0')
```

Comment:
I'm going to merge a PR that updates the dev dependencies rather than locking it to an older version of rake. Could you update the PR to remove the change to the gemspec?

Reply:
I added this to get travis ci to pass.  I removed it from my PR.